File: baekjoon/C++/3003.cpp
#include <iostream>

using namespace std;

void get_chess_count(int* arr)
{
  int count_arr[6] = {1, 1, 2, 2, 2, 8};
  
  int i;
  for(i = 0; i < 6; i++)
  {
    arr[i] = count_arr[i] - arr[i];
  }
}

int main()
{
  int i, chess_arr[6];
  
  for(i = 0; i < 6; i++)
  {
    cin >> chess_arr[i];
  }
  
  get_chess_count(chess_arr);
  
  for(i = 0; i < 6; i++)
  {
    cout << chess_arr[i] << ' ';
  }
  
  return 0;
}

File: baekjoon/JAVA/3003.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        int[] chess_arr = new int[6];
        Scanner sc = new Scanner(System.in);

        for (int i = 0; i < 6; i++) {
            chess_arr[i] = sc.nextInt();
        }

        get_chess_count(chess_arr);

        for (int i = 0; i < 6; i++) {
            System.out.print(chess_arr[i]);
            System.out.print(' ');
        }
    }

    public static void get_chess_count(int[] arr) {
        int count_arr[] = { 1, 1, 2, 2, 2, 8 };

        for (int i = 0; i < 6; i++) {
            arr[i] = count_arr[i] - arr[i];
        }
    }
}